uuid.go

```diff
@@ -151,6 +151,11 @@ var (
 	NamespaceX500 = Must(FromString("6ba7b814-9dad-11d1-80b4-00c04fd430c8"))
 )
 
+// IsZero returns if the UUID is equal to the nil UUID
+func (u UUID) IsZero() bool {
```

Comment:
How about `IsNil`?

Reply:
This makes sense, updated the method name to IsNil.

Thanks!